### DISCUSSION TITLE
Can I install 2 agents on 1 device and also the 2 agents will be running on 2 diffrent servers or [Dual-Server Solution and Dual agent]

### DISCUSSION THREAD

**Discussion by bolavefasfas** [Ylianst/MeshCentral#6134] [Q&A] [answered]
- **Existing Configuration**: Currently running a MeshCentral server on a VPS, with an agent installed on our devices.
- **The Challenge**: Experiencing high latency (over 200 ping), primarily due to the distance between the VPS and our devices.
- **Proposed Solution**: Installing a second MeshCentral server on our home server, which has a much lower ping time (4-7 seconds).
- **The Query**:  Is it's possible to install this second agent (home server-based) on the same Windows PC where the first agent (VPS-based) is already installed. Also, will this setup work if we use the Tailscale IP and not a domain for the home server-based agent?
- **The Objective**: Our primary goal is to achieve a faster response time, enhancing the overall user experience.
- **Home Server Constraint**: A potential limitation is the inconsistent electricity supply at our home, which may result in the server not being operational 24/7.
- **Security Fortification**: I have SSL and an authenticator enabled on our VPS-hosted MeshCentral server. I want to bolster the security of our setup what more can I do.

This software is just amazing.

**Comment by si458** [comment, marked as answer]
yes just customise the meshagent - https://ylianst.github.io/MeshCentral/meshcentral/customization/#agent-branding
change the companyname, servicename, filename
then when u setup a 2nd server, and download its exe, its different to your first server

**Reply by bolavefasfas** [reply]
Thanks a lot will test it today.
I ahve customized my agent 1 
will do for agent 2 as well.
